thrust_model:
   k1: 52.3
   k2: 0
   k3: 0